bob/bobfile/bobfile.go

```diff
@@ -182,6 +216,24 @@ func initializeDependencies(dir string, task bobtask.Task, bobfile *Bobfile) []n
 	return nix.UniqueDeps(taskDeps)
 }
 
+func newRemotestore(endpoint *url.URL) (s store.Store) {
+	const sep = "/"
+
+	parts := strings.Split(strings.TrimLeft(endpoint.Path, sep), sep)
+
+	username := parts[0]
+	project := strings.Join(parts[1:], sep)
+
+	s = remotestore.New(
+		username,
+		project,
+		remotestore.WithClient(
+			storeclient.New("http://"+endpoint.Host),
```

Comment:
That should be https by default. We should add a explicit option to allow insecure.

Reply:
Added.